scripts/apps/authoring/authoring/directives/SendItem.js

```diff
@@ -50,6 +51,30 @@ export function SendItem($q, api, desks, notify, authoringWorkspace,
             scope.$watch(send.getConfig, activateConfig);
 
             scope.publish = function() {
+                const draftjsFieldsHavingUnresolvedSuggestions = Object.keys(scope.item)
+                    .filter((key) => {
+                        const fieldValue = scope.item[key];
+                        const isDraftjsField = Array.isArray(fieldValue)
+                            && fieldValue.length === 1
+                            && typeof fieldValue[0].blocks === 'object';
+
+                        if (isDraftjsField === false) {
+                            return false;
+                        }
+
+                        const rawState = fieldValue[0];
+
+                        return hasUnresolvedSuggestions(rawState);
```

Comment:
maybe it would better api if we have `hasUnresolvedSuggestions(scope.item)` and handle editor state inside.

Reply:
Yeah, I was thinking about this one myself. I want to keep `text-highlights` implementation modular and agnostic to our business logic e.g. the fields of the news item. Does it work if I put the logic to `authoring/helpers` as per 6346e6d ?